src/plugins/intel_cpu/src/nodes/kernels/scaled_attn/executor_pa.cpp

```diff
@@ -1205,10 +1205,10 @@ void rotate_kv_cache(PlainTensor& key_cache,
 template <typename DATA_TYPE, typename KEY_CACHE_TYPE, ov::element::Type_t VALUE_PREC>
 struct MHAHelper {
     // initialize once
-    size_t _H;
-    size_t _S;
-    size_t _SV;
-    size_t _Hk;
+    size_t H;
```

Comment:
What check is asking to change this?

Reply:
Discussed offline, it is [bugprone-reserved-identifier](https://clang.llvm.org/extra/clang-tidy/checks/bugprone/reserved-identifier.html)